scripts/run_acceptance_tests.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2023 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Python execution for running acceptance tests."""
+
+from __future__ import annotations
+
+import argparse
+import contextlib
+import os
+import subprocess
+import sys
+
+from typing import Final, List, Optional, Tuple
+
+from scripts import common  # isort:skip pylint: disable=wrong-import-position, unused-import
+
+from core.constants import constants  # isort:skip
+from scripts import build  # isort:skip
+from scripts import servers  # isort:skip
+
+GOOGLE_APP_ENGINE_PORT: Final = 8181
+ELASTICSEARCH_SERVER_PORT: Final = 9200
+PORTS_USED_BY_OPPIA_PROCESSES: Final = [
+    GOOGLE_APP_ENGINE_PORT,
+    ELASTICSEARCH_SERVER_PORT,
+]
```

Comment:
These should not be redefined (I assume these are also defined in e2e runner).

Reply:
@ashish-patwal Did you see this comment? It has not been replied to.

Reply:
Ok @seanlip Made it common for all by sending it to common.py